src/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JS Capstone - Entertaiment Opinion</title>
+</head>
+<body>
+  <!-- Start header -->
+  <header>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-success">
+      <div class="container-fluid container p-4">
+        <img src="./images/icon.png" alt="">
+        <a class="navbar-brand" href="#"><h2>Movies</h2></a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+            <p>more</p>
+            </li>
+          </ul>
+          <form class="d-flex" role="search">
+            <input
+              class="form-control me-2"
+              type="search"
+              placeholder="place your command..."
+              aria-label="Search"
+            />
+            <button class="btn btn-danger" type="submit">Search</button>
+          </form>
+        </div>
+      </div>
+    </nav>
```

Comment:
- [ ] The `Header` looks great, but let's add a counter (total) of the available movies to match the wireframe we are given.

![total](https://user-images.githubusercontent.com/53124548/218267008-e80256f6-29d3-4366-8fc2-7ab679e88bf3.png)

![home](https://user-images.githubusercontent.com/53124548/218266980-4196e705-63c6-42c8-91b9-efbca96e1896.png)